scripts/python/testcases/tcTrigger.py

```diff
@@ -0,0 +1,253 @@
+import api.tutils
+import time
+import sys
+import api.objmodel
+import random
+
+# Check ping between all containers in the same network
+def checkPingContainersInNetworks(containers, networks):
+    cntrIpList = []
+    # Read all IP addresses
+    for cnt in containers:
+        cntrIp = cnt.getIpAddr()
+        cntrIpList.append(cntrIp)
+
+    nidx = random.randint(0, (len(networks) - 1))
```

Comment:
It is generally possible to reproduce an issue. Here is the thinking: assume there are `n` triggers, there is `n*(n-1)` different combination of triggers possible. We run thru `K * n*(n-1)` iterations in a random order to make sure every possible combination gets tested multiple times. So, its unlikely we'll hit an issue that is not possible to reproduce. 

If we start seeing unreproducible issues here, we can change the random order to a fixed order which tests all possible combinations.